### PR TITLE
Pagination added to Absence and Meeting

### PR DESCRIPTION
**Pull request by AnkushMalik**
**Meeting#index :**

**Absences#index :**

**changes in this PR :**
- Partials used for rendering code in meeting and absences index.
- pagination added to absences and meetings index using smart_listing.

### CODE REVIEW

File: app/views/meetings/_form.html.slim [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+    = f.input :event_type, collection: Meeting::TYPES, input_html: { class: 'ui fluid dropdown' }
+  .three.fields
+    = f.input :location, as: :string
+    = f.input :start_date, as: :string, wrapper_html: { class: 'calendar' }
+    = f.input :end_date, as: :string, wrapper_html: { class: 'calendar' }
+  = f.input :description, input_html: { rows: 3, cols: 40 }
+  .ui.right.floated.buttons
+    = link_to 'Cancel', meetings_path, class: 'ui button'
+    .or
+    = f.submit 'Create', class: 'ui green button'
```
Comment (DavidKang) [inline app/views/meetings/_form.html.slim]: Take into account that in this case you are editing a meeting a not creating it.

File: app/views/absences/_form.html.slim [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+  .two.fields
+    = f.input :start_date, as: :string,
+                           wrapper_html: { class: 'calendar_range_start' }
+    = f.input :end_date, as: :string,
+                         wrapper_html: { class: 'calendar_range_end' }
+  = f.input :description, input_html: { rows: 3, cols: 40 }
+  .ui.right.floated.buttons
+    = link_to 'Cancel', absences_path, class: 'ui button'
+    .or
+    = f.submit 'Create', class: 'ui green button'
```
Comment (DavidKang) [inline app/views/absences/_form.html.slim]: Same here

File: app/views/meetings/_form.html.slim (hunk 1 above)
Comment (AnkushMalik) [reply app/views/meetings/_form.html.slim]: ok, i will change the btn text to 'submit' (?)

Comment (DavidKang) [reply app/views/meetings/_form.html.slim]: @AnkushMalik, What about removing the text? 🤔

Comment (AnkushMalik) [reply app/views/meetings/_form.html.slim]: @DavidKang, As all other forms use text in btns ,using icon for a single form might looks a little weird.
contional btn text can easily be used by using 'action_name'

Comment (AnkushMalik) [reply app/views/meetings/_form.html.slim]: @DavidKang,its ok but as all other forms use text in btns ,using icon for a single form might look a little weird. :sweat_smile: 
contional btn text can easily be used by using 'action_name'
`= f.submit action_name == 'index' ? 'Create':'Update', class: 'ui green button'`

**Comment by AnkushMalik** [comment]
@DavidKang, I have made the changes :)

File: app/views/meetings/_form.html.slim [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
+    = f.input :event_type, collection: Meeting::TYPES, input_html: { class: 'ui fluid dropdown' }
+  .three.fields
+    = f.input :location, as: :string
+    = f.input :start_date, as: :string, wrapper_html: { class: 'calendar' }
+    = f.input :end_date, as: :string, wrapper_html: { class: 'calendar' }
+  = f.input :description, input_html: { rows: 3, cols: 40 }
+  .ui.right.floated.buttons
+    = link_to 'Cancel', meetings_path, class: 'ui button'
+    .or
+    = f.submit action_name == 'index' ? 'Create':'Update', class: 'ui green button'
```
Comment (DavidKang) [inline app/views/meetings/_form.html.slim]: Honestly, I don't like to use a ternary operation only to switch the text of a submit button. Why not change the local file (en.yml) and change the text of the submit button? 🤔

File: app/assets/javascripts/custom.coffee [hunk 4]
```diff
@@ -13,10 +13,8 @@ $(document).on "turbolinks:load", ->
     firstDayOfWeek: 1
 
   $('.ui.accordion').accordion()
-  $('.ui.dropdown').dropdown
-    action: 'hide'
 
-  $('select.dropdown').dropdown()
+  $('select.dropdown, .ui.dropdown').dropdown()
```
Comment (DavidKang) [inline app/assets/javascripts/custom.coffee]: This change is not related to with this PR, right?

Comment (AnkushMalik) [reply app/assets/javascripts/custom.coffee]: yes, it is. line:17's action: hide was causing an error to the dropdown menu  after I arranged dropdown's overflow in the forms.
